packages/azure-services/src/azure-batch/batch-task-config-generator.ts

```diff
@@ -6,11 +6,6 @@ import { inject, injectable } from 'inversify';
 import { cloneDeep } from 'lodash';
 import moment from 'moment';
 
-export interface BatchTaskPropertyProvider {
```

Comment:
This interface is required to have an optional implementation of included methods in derived classes. Why we remoning it?